### ISSUE TITLE
[1.10.2 to 1.11.2 conversion - 0.11-62] Converting to 1.11.2 makes Excavator bucket wheel invisible

### ISSUE THREAD

**Issue by VallenFrostweaver** [BluSunrize/ImmersiveEngineering#2206]
I did an update of my 1.10.2 IE tutorial world to 1.11.2 along with several mods with it. Upon entering I saw the bucket wheel of the Excavator was invisible. It's simple to break a block and replace to fix but I thought it was worth a mention.

EDIT: Breaking block (all reappears) and replacing keeps it invisible after using the hammer to convert to a multiblock so it is a bigger deal than expected.

 ![ie bucket wheel](https://user-images.githubusercontent.com/13642054/27428622-914cadc2-573a-11e7-9f7a-59cba301b5f8.png)

**Comment by malte0811** [comment]
What other mods are installed? I updated my dev world from 1.10 to 1.11 and don't remember having had issues with bucket wheels.

**Comment by VallenFrostweaver** [comment]
![mod list](https://user-images.githubusercontent.com/13642054/27430070-0e48c050-573f-11e7-825b-ccc1f6a23fc1.png)
This should help.

**Comment by VallenFrostweaver** [comment]
UPDATE: I did try a relog a few times to see if that was the issue but nothing changed. I since found the charging station and railgun errors and the bucket wheels are now visible. I have no clue here what happened. I did notice that while discovering issue #2208 that I could also see some textures changing in the background of some of my other machines and such. It's possible that this may have somehow started some textures to load in or something. Man I wish I knew how to code so that I could help better. :/
